### PR TITLE
Added schedule table in volunteer profile page

### PR DESCRIPTION
**Pull request by satwik-talluri**
# Related Issue
Fixes: #240

# Prosposed Changes
- Add schedule for volunteers in the form of a table in the profile section

# Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
Volunteer_initial  |  Volunteer_final

### CODE REVIEW

**Comment by satwik-talluri** [comment]
I have not made any changes in context_processors.py file But only changed views.py and profile.html.
I am still getting a flake8 error.

@garg3133 is it okay to merge?

File: apps/volunteers/views.py [hunk 1]
```diff
@@ -40,9 +43,17 @@ def index(request):
 )
 def profile(request, pk):
     volunteer = get_object_or_404(Volunteer, id=pk)
+    user=request.user
+    profile = Profile.objects.filter(user=user).first()
+    if profile is not None and user.desig == 'v':
+        volun = Volunteer.objects.get(profile=profile)
+    if volun is not None:
+        volun_schs = VolunteerSchedule.objects.all().filter(volun=volun).order_by('day')
+
```
Comment (garg3133) [inline apps/volunteers/views.py:52]: Why have you just copy pasted the code of `context_processors.py` here? Most of this code is not needed here.

File: apps/volunteers/views.py [hunk 2]
```diff
@@ -10,6 +10,9 @@
 # from django.template.loader import render_to_string
 from django.urls import reverse_lazy
 
+from accounts.models import Profile
+from apps.volunteers.models import Volunteer, VolunteerSchedule
+
```
Comment (garg3133) [inline apps/volunteers/views.py:15]: First import is not required and the second is duplicate.

File: apps/volunteers/views.py [hunk 3]
```diff
@@ -40,9 +43,17 @@ def index(request):
...
+    profile = Profile.objects.filter(user=user).first()
+    if profile is not None and user.desig == 'v':
+        volun = Volunteer.objects.get(profile=profile)
+    if volun is not None:
+        volun_schs = VolunteerSchedule.objects.all().filter(volun=volun).order_by('day')
+
     context = {
         'volunteer': volunteer,
         'self_profile': volunteer.profile.user == request.user,
+        'volun_schs': volun_schs,
```
Comment (garg3133) [inline apps/volunteers/views.py:56]: Just keep it `volun_sch` instead of `volun_sch`.

File: templates/volunteers/profile.html [hunk 4]
```diff
@@ -237,4 +237,42 @@ <h2>
         </div>
     </div>
 </div>
+
+<div class="container">
```
Comment (garg3133) [inline templates/volunteers/profile.html:241]: You don't need to create a new container element here. Just use the previous container and start with a new row.

**Review by garg3133 — COMMENTED** [review]
@satwik-talluri I think the card would look even better if we join the two cards (no margin between them) and then overlap a small bottom part of the first card over the second card.

**Comment by satwik-talluri** [comment]
hey @garg3133 , I have taken all the above suggestions into account.
Is this okay?
volunteer

**Comment by garg3133** [comment]
@satwik-talluri No, I meant two cards only, joined together with the bottom of the first overlapping the last. You can refer to the contact us section on the new home page (http://localhost:8000/new) on screen size < 768px.

**Comment by satwik-talluri** [comment]
@garg3133 Is this okay?
overlap

**Comment by garg3133** [comment]
@satwik-talluri I cannot test your changes. Have you deleted your fork after opening this PR?

**Comment by satwik-talluri** [comment]
> @satwik-talluri I cannot test your changes. Have you deleted your fork after opening this PR?

@garg3133  I did not open a PR for this yet, was waiting for the confirmation.

**Comment by garg3133** [comment]
> > @satwik-talluri I cannot test your changes. Have you deleted your fork after opening this PR?
> 
> @garg3133 I did not open a PR for this yet, was waiting for the confirmation.

You don't need to open PR everytime you make some changes. Just keep pushing to the same branch and it will get updated here.

And I'm not able o understand why you keep on deleting your fork? Its okay if your fork is behind by some commits, it's always gonna happen. You can update your fork by running 
```
git checkout master
git pull upstream master
git push origin master
```

But, mostly it doesn't cause any issue if your fork is behind. You don't need to delete and create a new fork everytime it happens.

**Comment by tushhr** [comment]
@satwik-talluri?

**Comment by satwik-talluri** [comment]
> @satwik-talluri as you already opened #247, I'm closing this.

Sure

**Pull request closed without merging**